src/Migrations/RefToInoutMigration.hack

```diff
@@ -158,14 +186,31 @@ final class RefToInoutMigration extends BaseMigration {
     return $n['root'];
   }
 
+  /**
+   * Same as above, but doesn't throw if the argument is not by-ref or inout.
```

Comment:
nit: replace `above` with method name in case of re-ordering or new functions being added.